.ci_support/linux_aarch64_.yaml

```diff
@@ -17,7 +17,7 @@ cxx_compiler:
 cxx_compiler_version:
 - '12'
 docker_image:
-- quay.io/condaforge/linux-anvil-aarch64
```

Comment:
This change looks wrong. The file is named as linux with `aarch64` architecture.

Reply:
@eaaltonen, I think this is correct, and reflects the change introduced by c7a8e34bba7cbab013b114d8e010d252ca2bb574 to cross-compile for `aarch64` (and `ppc64le`) on `x86_64`, so the docker image for the build is now the `cos7-x86_64`  image.

Reply:
Indeed, that's exactly what's happening. :)